.github/workflows/builds.yml

```diff
@@ -30,6 +30,9 @@ jobs:
           - name: CPU
             container: ghcr.io/acts-project/ubuntu2004:v30
             options: -DTRACCC_USE_ROOT=FALSE
+          - name: HIP 
+            container: ghcr.io/acts-project/ubuntu2204_rocm_clang
```

Comment:
You need to put a particular version number here. We can't have the CI using the "latest" image. (Allowing the CI to break by somebody updating the image in some unforeseen way.)

Reply:
My understanding is that we need a new tag of `machines` for this. I could change this to `ubuntu2204_rocm_clang:sha-3a6b0b2` for now, which matches latest. Or we can wait for a new machines tag.

Reply:
Then we should make a new tag of it. Though again, I believe that once we turn off the unit test builds of rocThrust, we could possibly survive with something as old as `ghcr.io/acts-project/ubuntu1804_rocm:v11`. 🤔

Reply:
Unfortunately not. It still insists on running its compiler checks with this change...